src/echoserver/EchoServer.java

```diff
@@ -9,23 +11,60 @@
 public class EchoServer {
 	
 	// REPLACE WITH PORT PROVIDED BY THE INSTRUCTOR
-	public static final int PORT_NUMBER = 0; 
+	public static final int PORT_NUMBER = 6013;
 	public static void main(String[] args) throws IOException, InterruptedException {
 		EchoServer server = new EchoServer();
 		server.start();
 	}
 
 	private void start() throws IOException, InterruptedException {
-		ServerSocket serverSocket = new ServerSocket(PORT_NUMBER);
-		while (true) {
-			Socket socket = serverSocket.accept();
-
-			// Put your code here.
-			// This should do very little, essentially:
-			//   * Construct an instance of your runnable class
-			//   * Construct a Thread with your runnable
-			//      * Or use a thread pool
-			//   * Start that thread
+		ServerSocket sock = new ServerSocket(PORT_NUMBER);
+		while(true){
+			//wait until someone accepts
+			Socket client = sock.accept();
+			System.out.println("HEY! We got somebody!");
+			//Input from the client
+			InputStream in = client.getInputStream();
```

Comment:
Not sure why you'd pull these out here instead of just passing the socket and let the `ServerInstance` class extract the streams.